Cargo.toml

```diff
@@ -1,16 +1,40 @@
 [package]
 name = "hades252"
-version = "0.1.0"
-authors = ["kev <kevtheappdev@gmail.com>"]
+version = "0.1.1"
+authors = [
+  "kev <kevtheappdev@gmail.com>", "zer0 <matteo@dusk.network>",
+  "Victor Lopes <victor@dusk.network>", "kr0 <c.perezbaro@gmail.com>"
```

Comment:
Should not be better to add all of the dusk mails?

Reply:
This is very true. :)
But since you and Kev used your personal e-mail for the commit; the `.toml` reflect that.

You both should configure your local environment to use a different author / email on dusk project, and configure github to recognize both emails (gmail and dusk) to your account.

Reply:
I thought I solved that vinculating both mails to the same GH account... 
Will take a look later.